src/analysis/signature_help.mli

```diff
@@ -0,0 +1,25 @@
+type parameter_info =
+  { label : Asttypes.arg_label
+  ; param_start : int
+  ; param_end : int
+  ; argument : Typedtree.expression option
+  }
+
+type application_signature =
+  { function_name : string option
+  ; function_position : Msource.position
+  ; signature : string
+  ; parameters : parameter_info list
+  ; active_param : int option
+  }
+
+val application_signature :
+     prefix:string
+  -> Mbrowse.t
+  -> application_signature option
+
+val prefix_of_position :
+    short_path: bool
+  -> Msource.t
+  -> Msource.position
+  -> string
```

Comment:
WDYT about moving (or duplicating) some of your meaningful comments from the `.ml` to the `.mli` as doc purpose?

Reply:
Moved